esrally/track/track.py

```diff
@@ -721,7 +721,24 @@ def __eq__(self, other):
         return isinstance(other, type(self)) and self.op_type == other.op_type
 
     def __str__(self, *args, **kwargs):
-        return "filter for operation type [%s]" % self.op_type
+        return f"filter for operation type [{self.op_type}]"
+
+
+class TaskTagFilter:
+    def __init__(self, tag_name):
+        self.tag_name = tag_name
+
+    def matches(self, task):
+        return self.tag_name in task.tags
```

Comment:
This appears to mean case sensitive matching? Is this what we want? I assume `include-tasks` is currently case sensitive?

Reply:
Correct, this behavior is consistent with current behavior, i.e. all other task filters are also case-sensitive.

Reply:
I pushed an additional note in the docs about case sensitivity of task filters in https://github.com/elastic/rally/pull/1154/commits/6cd13b22e6fce3dcd3acdc5a7bd6370af7957cd7.